examples/07_sglang_model.py

```
@@ -0,0 +1,54 @@
+    env={
+        "HF_HUB_ENABLE_HF_TRANSFER": "1",
+    },
+)
+def serve():
+    """Start SGLang server with Qwen model."""
+    import subprocess
+
+    cmd = [
+        "python3", "-m", "sglang.launch_server",
+        "--model-path", "Qwen/Qwen2.5-0.5B-Instruct",
+        "--host", "0.0.0.0",
```

Ruff (0.14.7)

Possible binding to all interfaces

(S104)